sync/requirements.txt

```diff
@@ -1,3 +1,3 @@
-boto3>=1.21.24
-cryptography>=36.0.2
-dataclasses-json>=0.5.7
+boto3>=1.21.24,<2
+cryptography>=36.0.2,<37
+dataclasses-json==0.5.7
```

Comment:
Why fix the version?

Reply:
Anything starting with 0.* might imply not stable backward compatible version. In that case we should stick to one version to secure ourselves from potential regressions.

Reply:
Even for 0.*, libraries stay consistent throughout their minor versions. So, it would make sense to have 0.5.x at least. Otherwise, we're going to stick to an old library version without updates